### PR TITLE
update kube verison in requirements

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**
- Update the kube version in the requirements

**Motivation for the change:**
https://github.com/operator-framework/getting-started/issues/60

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,8 +45,8 @@ The following workflow is for a new **Helm** operator:
 - [mercurial][mercurial_tool] version 3.9+
 - [docker][docker_tool] version 17.03+.
   - Alternatively [podman][podman_tool] `v1.2.0+` or [buildah][buildah_tool] `v1.7+`
-- [kubectl][kubectl_tool] version v1.11.3+.
-- Access to a Kubernetes v1.11.3+ cluster.
+- [kubectl][kubectl_tool] version v1.15.4+.
```
Comment (estroz) [inline README.md]: Technically SDK projects built with `operator-sdk` v0.12.0 are compatible with kube v1.12+.

This issue brings up our need for a compatibility matrix [again](https://github.com/operator-framework/operator-sdk/issues/2118).

Comment (camilamacedo86) [reply README.md]:
```suggestion
- [kubectl][kubectl_tool] version v1.12.0+.
```

File: README.md [hunk 2]
```diff
@@ -45,8 +45,8 @@ The following workflow is for a new **Helm** operator:
 - [mercurial][mercurial_tool] version 3.9+
 - [docker][docker_tool] version 17.03+.
   - Alternatively [podman][podman_tool] `v1.2.0+` or [buildah][buildah_tool] `v1.7+`
-- [kubectl][kubectl_tool] version v1.11.3+.
-- Access to a Kubernetes v1.11.3+ cluster.
+- [kubectl][kubectl_tool] version v1.12.0+.
+- Access to a Kubernetes v1.15.4+ cluster.
```
Comment (camilamacedo86) [inline README.md]:
```suggestion
- Access to a Kubernetes v1.12.0+ cluster.
```

File: README.md [hunk 3]
```diff
@@ -45,8 +45,8 @@ The following workflow is for a new **Helm** operator:
 - [mercurial][mercurial_tool] version 3.9+
 - [docker][docker_tool] version 17.03+.
   - Alternatively [podman][podman_tool] `v1.2.0+` or [buildah][buildah_tool] `v1.7+`
-- [kubectl][kubectl_tool] version v1.11.3+.
-- Access to a Kubernetes v1.11.3+ cluster.
+- [kubectl][kubectl_tool] version v1.12.0+.
+- Access to a Kubernetes v1.12.0+ cluster.
 - Optional: [delve](https://github.com/go-delve/delve/tree/master/Documentation/installation) version 1.2.0+ (for `up local --enable-delve`).
```
Comment (camilamacedo86) [inline README.md]: @estroz changed for 0.12 as we spoke about and tracked: https://github.com/operator-framework/operator-sdk/issues/2183

**Pull request merged**